test/primitives-test.coffee

```diff
@@ -146,6 +146,129 @@ describe 'Minim Primitives', ->
           foo: 'bar'
           z: 2
 
+  describe 'ElementType', ->
+    elType = undefined
+
+    before ->
+      elType = new minim.ElementType 'element'
+
+    describe '#getProperty', ->
```

Comment:
feels like this could really be DRYed out a lot:

```coffeescript
describe 'ElementType', ->
  elType = new minim.ElementType 'element'
  missingMethods =
     getProperty: elType.getProperty('foo')
     map: elType.map((el) -> 'test')
     # Etc

  for method, missingMethod of missingMethods
    describe "##{method}", ->
      expect(missingMethod.elementType()).to.equal('error')
```